app/controllers/posts_controller.rb

```diff
@@ -166,6 +166,22 @@ def toggle_comments
     render json: { success: true }
   end
 
+  def feature
+    data = {
+      label: @post.parent.nil? ? @post.title : @post.parent.title,
+      link: helpers.generic_show_link(@post),
+      post: @post,
+      active: true
+    }
+    @link = PinnedLink.create data
+
+    attr = @link.attributes.map { |k, v| "#{k}: #{v}" }.join(' ')
+    AuditLog.moderator_audit(event_type: 'pinned_link_create', related: @link, user: current_user,
+                            comment: "<<PinnedLink #{attr}>>\n(using moderator tools on post)")
+    flash[:success] = 'Post has been featured. Due to caching it may take some time, until the changes apply.'
```

Comment:
Switch the comma around: "Due to caching, it may take some time until the changes are applied."